### ISSUE TITLE
Unable to locate original post

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#8467]
### Expected behavior
Can comment.

### Actual behavior
Cannot comment anymore. I get a message 'unable to locate original post' instead.

### Steps to reproduce the problem
Go to e.g. one of your own posts. Try to comment.

### Friendica version you encountered the problem
2020.03-rc

### Friendica source (git, zip)
git

### PHP version
7.3

### SQL version
10.3 MariaDB

Caused by #8464

**Comment by AlfredSK** [comment]
@annando Must be something related to the latest commits. The error didn't happen yesterday.

**Comment by AlfredSK** [comment]
Yes. Found the PR that is causing it.

```
# git log
commit cb1696bfaf083e1941687d08343d776986d8c8a2 (HEAD -> 2020.03-rc, origin/2020.03-rc)
Merge: 1d8bb726d 1194f9a58
Author: Tobias Diekershoff <tobias.diekershoff@gmx.net>
Date:   Sun Mar 29 08:37:57 2020 +0200

    Merge pull request #8464 from annando/notice
    
    Fixes two notices
```

**Comment by AlfredSK** [comment]
Hopefully fixed by #8466

**Comment by AlfredSK** [comment]
Works again. :-)
